server/plugin/api.go

```diff
@@ -492,13 +486,15 @@ func (p *Plugin) getConnected(c *Context, w http.ResponseWriter, r *http.Request
 		GitHubClientID    string        `json:"github_client_id"`
 		EnterpriseBaseURL string        `json:"enterprise_base_url,omitempty"`
 		Organization      string        `json:"organization"`
-		Settings          *UserSettings `json:"settings"`
+		UserSettings      *UserSettings `json:"user_settings"`
+		PluginSettings    Settings      `json:"plugin_settings"`
```

Comment:
Seems weird that we're using a pointer for one and not the other

Reply:
2/5 pointers make writing predictable test cases difficult, hence didn't use for mine implementation. Didn't change old due to issue of unrelated change. Let me know If changes required. Thanks.